Jenkinsfile

```diff
@@ -5,19 +5,37 @@ pipeline {
             steps {
                 sh 'hatch run docs:build'
             }
+
+	    post {
+      		 success {
+        	  publishHTML([allowMissing: false,
+                     alwaysLinkToLastBuild: false,
+                     keepAll: true,
+                     reportDir: 'docs/site',
+                     reportFiles: 'index.html',
+                     includes: '**/*',
+                     reportName: 'Cycax Docs',
+                     reportTitles: 'Cycax Docs report'
+                  ])
+                }
+            }	
+        }
+	stage('Mypy') {
+	    steps {
+		sh (label: 'Run MyPy',
+		    script: '''
+			    mkdir -p logs
+			    hatch run lint:typing | tee logs/mypy.log
+			    '''
+		)
+	    }
+
+	    post{
+                always {
+                  recordIssues(tools: [myPy(name: 'MyPy', pattern: 'logs/mypy.log')])
+                  publishHTML([allowMissing: false, alwaysLinkToLastBuild: false, keepAll: false, reportDir: 'logs/site', reportFiles: 'index.html', reportName: 'MyPy Logs', reportTitles: 'Cycax Mypy logs'])
```

Comment:
publishHTML does not make sense here?

Reply:
Yes it does not, I saw it used somewhere but I guess they were generating more than reports in their commands.